### DISCUSSION TITLE
Sweep Trigger Control

### DISCUSSION THREAD

**Discussion by barisyilm** [jankae/LibreVNA#219] [Q&A]
Hi, I'd like to control this VNA using a trigger signal, every time trig get high sweep should start. This could be done with an external input or by using an embedded counter within the librevna.Is that possible? If it is can you help? Its frequency should be around 5 Hz.

**Comment by jankae** [comment]
Sorry, that isn't possible. The LibreVNA does not have a trigger input. You could try to capture the trigger signal with other hardware and use a script to control the LibreVNA. But there would likely by a lot of jitter and I am not sure whether you could achieve 5 Hz sweep update rate.
